src/components/bandCombos/BandComboSearchForm.js

```diff
@@ -11,7 +15,7 @@ class BandComboSearchForm extends Component {
       style: '',
       study_area: '',
       bird__status: '',
-      colours: '',
+      colours: [],
```

Comment:
Store as an array of objects to easily manipulate select options. Convert to CSV when creating query.